test/index.html

```diff
@@ -5,11 +5,12 @@
 <link rel="stylesheet" href="../build/test.css">
 <style>
   .fixture-Box {
-    background: rgb(100, 150, 50);
-    background: rgb(100, 150, 50);
-    border: 1px solid black;
+    background: rgba(100, 150, 50, .75);
+    border: 1px solid #888;
+    font-family: sans-serif;
     margin-bottom: 10px;
```

Comment:
Yeah not sure on spacing as part of the grid. How do other ones handle this?

I think this raises a wider issue I faced which is lack of a proper demo page for users. Like http://flexboxgrid.com/